Bio/KEGG/KGML/KGML_pathway.py

```diff
@@ -87,7 +90,7 @@ def get_KGML(self):
                 "<!-- Created by KGML_Pathway.py %s -->" % time.asctime(),
             ]
         )
-        rough_xml = header + _as_string(ET.tostring(self.element, "utf-8"))
+        rough_xml = header + as_string(ET.tostring(self.element, "utf-8"))
```

Comment:
Do we need the as_string here? It looks like elementtree is already returning a string.

Reply:
It may well be redundant now - worth exploring later?

Reply:
Let's do it now.

Reply:
I agree with @mdehoon on this, done.